env/includes/engine_locations.yml

```diff
@@ -82,7 +82,7 @@ engines.tk-shotgun.location:
 # Unreal
 engines.tk-unreal.location:
   type: git_branch
-  path: git@github.com:shotgunsoftware/tk-unreal
+  path: https://github.com/shotgunsoftware/tk-unreal.git
```

Comment:
@ryanmayeda @pscadding I really think we should switch to using version numbers (e.g. tags) for our config. This is aligned with how we release everything else, it's way easier to work with version numbers from a support point of view, and it makes it clear what to upgrade to. Is there anything preventing us from doing that?